book/guides/data_guide.md

```diff
@@ -17,21 +17,166 @@ To help onboard you to this new way of working, we have written a guide to Files
+:::
+
+(data.config-files)=
+
+#### Configuration for Authenticated Access
+
+Unless a given cloud bucket allows anonymous access or is preauthenticated within your environment (like it is the case for some of the [LEAP-Pangeo owned buckets](reference.infrastructure.buckets)) you will need to authenticate with a key/secret pair.
+
+:::\{admonition} Always Handle credentials with care!
+:class: warning
+Always handle secrets with care. Do not store them in plain text that is visible to others (e.g. in a notebook cell that is pushed to a public github repository). See [](guide.code.secrets) for more instructions on how to keep secrets safe.
+:::
+
+We recommend to store your secrets in one of the following configuration files (which will be used in the following example to read and write data):
+s
+
+`````{tab-set}
+````{tab-item} Fsspec
+Fsspec supports named [](aws profiles) in a credentials files. You can create one via Generate an aws credential file via the [aws CLI](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html#cli-configure-files-examples)(installed on the hub by defaule):
+
+```shell
+aws configure --profile <pick_a_name>
+```
+Pick a sensible name for your profile, particularly if you are working with multiple profiles and buckets.
+
+The file `~/.aws/credentials` then contains your key/secret similar to this:
+
+```
+[<the_profile_name_you_picked>]
+aws_access_key_id = ***
+aws_secret_access_key = ***
+```
+````
+
+````{tab-item} Rclone
+Rclone has its own [configuration file format](https://rclone.org/docs/#config-config-file) where you can specify the key and secret (and many other settings) in a similar fashion (note the missing `aws_` though!).
+
+We recommend setting up the config file (show the default location with `rclone config file`) by hand to look something like this:
+
+```
+[<remote_name>]
+... # other values
+access_key_id = XXX
+secret_access_key = XXX
+```
+
+You can have multiple 'remotes' in this file for different cloud buckets.
+
+For the [](reference.infrastructrue.osn_pod) use this remote definition:
+
+```
+[osn]
+type = s3
+provider = Ceph
+endpoint = https://nyu1.osn.mghpcc.org
+access_key_id = XXX
+secret_access_key = XXX
+```
+
+````
+`````
+
+:::\{warning}
+Ideally we want to store these secrets only in one central location. The natural place for these seems to be in an [AWS cli profiles](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html#cli-configure-files-format), which can also be used for fsspec. There however seem to be multiple issues ([here](https://forum.rclone.org/t/shared-credentials-file-is-not-recognised/46993)) around this feature in rclone, and so far we have not succeeded in using AWS profiles in rclone.
```

Comment:
I spent quite some time to try to reconcile this. Ideally we only want to store one config file (aws profile) and then reference that from the rclone config, but I did not manage to get this to work at all so far. Might be worth tracking in an issue.